inc/ticket.class.php

```diff
@@ -89,7 +89,7 @@ function ($carry, $type) use ($item) {
             ];
 
             if (!isset($item->input['_actors'])) {
-                $item->input['_actors'] = $item->fields['_actors'];
+                $item->input['_actors'] = $item->fields['_actors'] ?? '';
```

Comment:
`_actors` is supposed to be an array?
```suggestion
                $item->input['_actors'] = $item->fields['_actors'] ?? [];
```